andela/ah-the-answer-frontend
# #164829344 Ft implement comments history display

#### What does this PR do?
* Implements the ability of all users to view the history of an edited comment

#### Description of the task to be completed
* Enable all users to view the history of an edited comment
* Lists the full history of an edited comment including timestamps
#### How should this be manually tested
* Clone the repo and navigate to the root directory using a terminal
* Install all the dependencies by typing `npm install`
* Start the server by typing `npm start`
* Navigate to the main page using a browser at http://localhost:3000/
<img width="1440" alt="Screenshot 2019-05-16 at 19 52 50" src="https://user-images.githubusercontent.com/26567020/58156237-50830c80-7c7e-11e9-9ab2-58c0af1fead0.png">
* Sign up and log in
<img width="1440" alt="Screenshot 2019-05-16 at 19 53 01" src="https://user-images.githubusercontent.com/26567020/58156248-57118400-7c7e-11e9-9592-d46dce408ff6.png">
* Once on the homepage click an article to view its details. You should see a comment form with a button labelled `comment`
<img width="1440" alt="Screenshot 2019-05-16 at 19 53 49" src="https://user-images.githubusercontent.com/26567020/58156281-6a245400-7c7e-11e9-8f9b-5d7a77dc8e3f.png">

* Write your comment in the form and press the button
<img width="788" alt="Screenshot 2019-05-22 at 10 46 08" src="https://user-images.githubusercontent.com/26567020/58156523-f171c780-7c7e-11e9-9770-efcfeba3b9cd.png">

* Hover over the comment to see `edit` icon and click on it to edit the comment 
<img width="746" alt="Screenshot 2019-05-22 at 10 46 41" src="https://user-images.githubusercontent.com/26567020/58156540-ffbfe380-7c7e-11e9-98e3-1527c49d3563.png">

* When done click `update` you should see the updated comment with an `edit` link at the bottom left.
<img width="767" alt="Screenshot 2019-05-22 at 10 48 19" src="https://user-images.githubusercontent.com/26567020/58156648-3a298080-7c7f-11e9-89a4-dd2bd8d963e8.png">

* Click on the `edit` link to view the history
<img width="710" alt="Screenshot 2019-05-22 at 10 48 41" src="https://user-images.githubusercontent.com/26567020/58156662-4281bb80-7c7f-11e9-9da8-b26a8df4e151.png">
